h/static/styles/group-form.scss

```diff
@@ -105,6 +105,11 @@
 
     &__list li {
       margin-bottom: 1.5em;
+
+      em {
```

Comment:
For consistency with the other style names, I think this would be better with a span with a suitable class rather than `<em>`, eg. `.group-documents-list__doctype` (in the `.scss` file, that means moving the contents of the `em {}` block here to a `&__doctype {}` block)